content/projects/gsoc/2019/project-ideas/gitlab-support-for-multibranch-pipeline.adoc

```diff
@@ -17,6 +17,7 @@ mentors:
   id: "jeffpearce"
   github: "jeffpearce"
 links:
+  gitter: https://gitter.im/jenkinsci/gitlab-branch-source-plugin
```

Comment:
```suggestion
  gitter: jenkinsci/gitlab-branch-source-plugin
```

Reply:
Our macros form URL automatically

Reply:
Thank you for the review. Kindly do not merge this PR, there is another project that has added the URL. I will commit those changes to this PR as well.